atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/EteSetup.java

```diff
@@ -186,7 +190,10 @@ protected void before() throws Throwable {
 
         return AtlasDbHttpClients.createProxyWithFailover(
                 new MetricRegistry(),
-                ImmutableServerListConfig.builder().addAllServers(uris).build(),
+                ImmutableServerListConfig.builder()
+                        .addAllServers(uris)
+                        .sslConfiguration(SslConfiguration.of(Paths.get("var/security/trustStore.jks")))
```

Comment:
nit: pull out the `SslConfiguration` into a separate constant? then share it with `TRUST_CONTEXT` above?